### DISCUSSION TITLE
mention user功能疑问

### DISCUSSION THREAD

**Discussion by ELEVENStudio-Main** [hydro-dev/Hydro#217] [General]
at用户的语法和插入视频的语法貌似是一样的，请问是冲突了吗？

`按照@[xxx](/user/123)的语法只会显示/user/123，且不会收到通知。`

错了闭嘴

**Comment by wuxianucw** [comment]
这个是刻意为之的，但是目前并没有做好，所以不可用

**Comment by undefined-moe** [comment]
`@[user](/user/:id)` 是特殊处理的。 你可以编辑器中使用 @username 自动输入。
